packages/web/src/routes/settings.tsx

```diff
@@ -211,6 +224,53 @@ function Settings() {
           </label>
           <DirPicker id="base-dir-picker" dirname={baseDir} entries={entries} cta="Change" />
         </div>
+        <div>
+          <h2 className="text-xl pb-2">Get product updates</h2>
+          <div>
+            <label className="opacity-70 text-sm" htmlFor="mailing-list-email">
+              Subscribe to our mailing list to get the latest updates, early access features, and
+              expert tips delivered to your inbox.
+            </label>
+            <div className="flex gap-2 mt-4">
+              <Input
+                id="mailing-list-email"
+                type="text"
+                placeholder="Your email"
+                value={email}
+                onChange={(e) => setEmail(e.target.value)}
+              />
+              <Button
+                className="px-5"
+                onClick={async () => {
+                  try {
+                    const response = await subscribeToMailingList(email);
+                    if (response.success) {
+                      setSubscribedEmailStatus(email);
+                      setEmailStatus({ email, dismissed: false });
+                      toast.success('Subscribed successfully!');
+                    } else if (response.status === 409) {
```

Comment:
It'd be better to return success from the API when the user is already subscribed and then do away with this state which doesn't add anything IMO. This is especially true in this case because we hide the form afterwords and so they shouldn't typically have the ability to even do this.

Reply:
It's slightly more info (you're already on there) but happy to simplify I like simple.

Reply:
Done